src/EnergyPlus/EconomicTariff.cc

```diff
@@ -482,11 +485,67 @@ namespace EconomicTariff {
 				tariff( iInObj ).energyConv = 9.4781712e-9;
 				tariff( iInObj ).demandConv = 0.00003412;
 			} else {
-				tariff( iInObj ).convChoice = conversionKWH;
-				tariff( iInObj ).energyConv = 0.0000002778;
-				tariff( iInObj ).demandConv = 0.001;
-				ShowWarningError( RoutineName + CurrentModuleObject + "=\"" + cAlphaArgs( 1 ) + "\" invalid data" );
-				ShowContinueError( cAlphaFieldNames( 3 ) + "=\"" + cAlphaArgs( 3 ) + "\", Defaulting to KWH." );
+
+
+				// TODO: is this needed?
```

Comment:
@JasonGlazer You make a good point.  The code knows the base units for a given meter, so it can just be smart enough to apply a different factor for CCF on a water meter vs CCF on a gas meter.  And CCF on an electric meter should be an error.  So, this can be fixed without any IDD changes and update the docs.  If we want to go a step further, then we could also add gallons and 1000gallons, but that could wait.